### PR TITLE
doc: update three figures with brand-compliant designs

### PR DESCRIPTION
**Pull request by Avalei**
Updated the figures to use brand-approved colors and fonts.

Ref. [TECHDOC-1982](https://projecttools.nordicsemi.no/jira/browse/TECHDOC-1982)

Signed-off-by: Deidre Casey <deidre.casey@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/images/bt_mesh_device_lifecycle.svg [hunk 1]
```diff
@@ -2,93 +2,89 @@
...
-		<g id="shape19-1" transform="translate(0.12,-78.7429)">
-			<title>Process</title>
+		<g id="shape1032-1" transform="translate(140.093,-146.592)">
+			<title>Nordic Blue.1032</title>
+			<desc>Provisioned mesh network mode</desc>
+			<rect x="0" y="262.979" width="102.829" height="49.9973" class="st1"/>
+			<text x="8.16" y="284.38" class="st2">Provisioned mesh <tspan x="15.54" dy="1.2em" class="st3">network mode</tspan></text>		</g>
+		<g id="shape1033-5" transform="translate(279.186,-81.0407)">
+			<title>Nordic Blue.1033</title>
+			<desc>Configure mesh network mode</desc>
```
Comment (ru-fu) [inline doc/nrf/images/bt_mesh_device_lifecycle.svg:40]: The original image said "Configured mesh network node"

File: doc/nrf/images/bt_mesh_device_lifecycle.svg [hunk 2]
```diff
@@ -2,93 +2,89 @@
...
+			<use xlink:href="#lend13" transform="scale(-3.52,-3.52) "/>
 		</marker>
 	</defs>
 	<g>
 		<title>Page-1</title>
-		<g id="shape19-1" transform="translate(0.12,-78.7429)">
-			<title>Process</title>
+		<g id="shape1032-1" transform="translate(140.093,-146.592)">
+			<title>Nordic Blue.1032</title>
+			<desc>Provisioned mesh network mode</desc>
```
Comment (ru-fu) [inline doc/nrf/images/bt_mesh_device_lifecycle.svg:35]: The original image said node, not mode

**Review by ru-fu — CHANGES_REQUESTED** [review]
These look a lot nicer now! 👍 

But some issues with the text for the mesh one, unless you wanted to change it?

Also, can you check in the Visio source files as well?

File: doc/nrf/images/bt_mesh_device_lifecycle.svg (hunk 1 above)
Comment (Avalei) [reply doc/nrf/images/bt_mesh_device_lifecycle.svg]: Good catch, this is now fixed.

File: doc/nrf/images/bt_mesh_device_lifecycle.svg (hunk 2 above)
Comment (Avalei) [reply doc/nrf/images/bt_mesh_device_lifecycle.svg]: Good catch, this is now fixed.

**Comment by Avalei** [comment]
Fixed typos from "mode" to "node, and "configure" to "configured". Uploaded the visio (.vsdx) file.

**Review by ru-fu — APPROVED** [review]
Thanks!

**Review by mia-ko — APPROVED** [review]
Looks much better!

**Pull request merged**